src/components/PlantsV2/components/TotalReportedPlantsCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import strings from 'src/strings';
+import { Box, Typography, useTheme } from '@mui/material';
+import { useAppSelector } from 'src/redux/store';
+import { selectSitePopulation } from 'src/redux/features/tracking/sitePopulationSelector';
+
+type TotalReportedPlantsCardProps = {
+  plantingSiteId?: number;
+};
+
+export default function TotalReportedPlantsCard({ plantingSiteId }: TotalReportedPlantsCardProps): JSX.Element {
+  const theme = useTheme();
+  const populationSelector = useAppSelector((state) => selectSitePopulation(state));
+  const [totalPlants, setTotalPlants] = useState(0);
+  useEffect(() => {
+    if (populationSelector) {
+      const sum =
+        populationSelector?.reduce((acc1, zone) => {
+          const sum1 = Number(
+            zone.plantingSubzones?.reduce((acc2, sz) => {
+              const sum2 = Number(
+                sz.populations?.reduce((acc3, pop) => {
+                  return acc3 + pop.totalPlants;
+                }, 0)
+              );
+              return acc2 + (isNaN(sum2) ? 0 : sum2);
+            }, 0)
+          );
+          return acc1 + (isNaN(sum1) ? 0 : sum1);
+        }, 0) ?? 0;
+      setTotalPlants(sum);
+    }
+  }, [populationSelector]);
+
+  const numberFontSize = (n: number): string => {
```

Comment:
At the narrowest point (just before the tablet->mobile breakpoint) the number can overflow the card. Dynamically adjust the font size to avoid this
<img width="242" alt="image" src="https://github.com/terraware/terraware-web/assets/114949086/3df3cec5-309b-48c6-b644-25a8d8b018a1">
<img width="239" alt="image" src="https://github.com/terraware/terraware-web/assets/114949086/32d9dd4b-7302-4162-b786-0b8f45ba7f35">
<img width="237" alt="image" src="https://github.com/terraware/terraware-web/assets/114949086/14a762e9-80d8-4c7d-9d59-53d7ebb27154">
etc.